### DISCUSSION TITLE
2007/05/22/%E7%BE%8E%E5%9B%BD%E6%9C%89%E4%BA%9B%E5%95%86%E5%9C%BA%E5%88%B7%E4%BF%A1%E7%94%A8%E5%8D%A1%E6%98%AF%E7%9B%B4%E6%8E%A5%E4%BD%BF%E7%94%A8%E4%BA%BA%E6%B0%91%E5%B8%81%E7%BB%93%E7%AE%97%E7%9A%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#162] [Announcements]
# 2007/05/22/%E7%BE%8E%E5%9B%BD%E6%9C%89%E4%BA%9B%E5%95%86%E5%9C%BA%E5%88%B7%E4%BF%A1%E7%94%A8%E5%8D%A1%E6%98%AF%E7%9B%B4%E6%8E%A5%E4%BD%BF%E7%94%A8%E4%BA%BA%E6%B0%91%E5%B8%81%E7%BB%93%E7%AE%97%E7%9A%84

我在美国一直使用我国内的双币信用卡，一直是按美元结算的。前些天我帮人在 Dillard's 买了一个化妆品，一看收银条，就然写着人民币多少多少钱。几天后查信用卡的记录，果然是直接从人民币账户扣款的。它的汇率是按照7.9几计算的，不知对我们用户来说是否划算。

https://blog.qizhen.xyz/2007/05/22/%E7%BE%8E%E5%9B%BD%E6%9C%89%E4%BA%9B%E5%95%86%E5%9C%BA%E5%88%B7%E4%BF%A1%E7%94%A8%E5%8D%A1%E6%98%AF%E7%9B%B4%E6%8E%A5%E4%BD%BF%E7%94%A8%E4%BA%BA%E6%B0%91%E5%B8%81%E7%BB%93%E7%AE%97%E7%9A%84

**Comment by ruanqizhen** [comment]
之前博客下的留言：

kat
2007年06月3日 22:38 编辑
应该是按照交易当日的汇率来结算的吧？

回复
俊
2007年05月23日 09:31 编辑
当然亏大了啊，现在的汇率才7.6几啊
